runtime/jvmti/j9jvmti.tdf

```diff
@@ -666,3 +666,15 @@ TraceExit-Exception=Trc_JVMTI_loadAgentLibraryGeneric_agentAttachFailed1_Exit Ov
 TraceExit-Exception=Trc_JVMTI_loadAgentLibraryGeneric_agentAttachFailed2_Exit Overhead==1 Level=1 Noenv Template="loadAgentLibraryGeneric failed attaching agent library %s; load function %s failed with error code=%d"
 TraceExit=Trc_JVMTI_loadAgentLibraryGeneric_succeed_Exit Overhead=1 Level=1 Noenv Template="loadAgentLibraryGeneric successfully loaded library %s and function %s"
 TraceEvent=Trc_JVMTI_isAgentLibraryLoaded_result Overhead=1 Level=5 Noenv Template="isAgentLibraryLoaded() library (%s) agentPath (%s) result (%zu)"
+TraceEvent=Trc_JVMTI_isAgentLibraryLoaded_library_decorate Overhead=1 Level=5 Noenv Template="isAgentLibraryLoaded() library(%s) decorate(%zu)"
+TraceEvent=Trc_JVMTI_loadqueryGetObjectFiles_OOM Overhead=1 Level=1 Noenv Template="loadqueryGetObjectFiles() OOM with objectFileSize(%zu)"
+TraceEvent=Trc_JVMTI_isAgentLibraryLoaded_OOM Overhead=1 Level=1 Noenv Template="isAgentLibraryLoaded() OOM at %s"
+TraceEvent=Trc_JVMTI_loadqueryGetObjectFiles_loadquery_errno Overhead=1 Level=1 Noenv Template="loadqueryGetObjectFiles:loadquery() failed with errno(%zd)"
+TraceEvent=Trc_JVMTI_loadqueryGetObjectFiles_allocate_memory Overhead=1 Level=5 Noenv Template="loadqueryGetObjectFiles() tmpBuffer(%p) objectFileSize(%zu)"
+TraceEvent=Trc_JVMTI_isAgentLibraryLoaded_libNames Overhead=1 Level=5 Noenv Template="isAgentLibraryLoaded() libNameSo = %s, libNameAr = %s"
+TraceEvent=Trc_JVMTI_isAgentLibraryLoaded_loadquery_getlibpath Overhead=1 Level=5 Noenv Template="isAgentLibraryLoaded:loadquery(L_GETLIBPATH) returns libpath(%s) libpathSize(%zu)"
+TraceEvent=Trc_JVMTI_isAgentLibraryLoaded_ldinfo_filename_decorate Overhead=1 Level=5 Noenv Template="isAgentLibraryLoaded() ldinfo_filename (%s) decorate(%zu)"
+TraceEvent=Trc_JVMTI_isFileNameInLibPath_path_not_match Overhead=1 Level=5 Noenv Template="isFileNameInLibPath() fileName(%s) path doesn't match (%s) with dirLen (%zu)"
+TraceEvent=Trc_JVMTI_isFileNameInLibPath_path_match Overhead=1 Level=3 Noenv Template="isFileNameInLibPath() fileName(%s) path matches (%s) with dirLen (%zu)"
```

Comment:
Please use consistent templates; most enclose values in parentheses without a space before the opening parenthesis.
```
TraceEvent=Trc_JVMTI_isAgentLibraryLoaded_result Overhead=1 Level=5 Noenv Template="isAgentLibraryLoaded() library(%s) agentPath(%s) result(%zu)"
```
The format for `errno` should be `%d` (not `%zd`):
```
TraceEvent=Trc_JVMTI_loadqueryGetObjectFiles_loadquery_errno Overhead=1 Level=1 Noenv Template="loadqueryGetObjectFiles:loadquery() failed with errno(%d)"
```